devlog/2026-08-28-2036-decision-surface-identity.md

```
@@ -0,0 +1,168 @@
+# Decision-Surface Identity (Plan Revision 43)
+
+**Work unit:** #942, contract chain head of Wave 7 (#1001). **Origin:**
+#916 fixed the four invariants a recommendation source record's commitment
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)